CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributor Guide
+
+## Setting up a KFServing Environment
+TODO flesh this out
+1. make all
+2. make docker-build
+3. make docker-push
+4. make deploy
+5. make install
+4. kubectl apply -f docs/samples/tensorflow.yaml
```

Comment:
Believe this should be part of developer guide, and we should follow the general convention for CONTRIBUTING guidelines for rest of the repos here.

Reply:
SGTM